guest-os-info.md

```diff
@@ -0,0 +1,35 @@
+## Guest Operating System Information
+
+Guest operating system identity for the Virtual Machine will be provided by the label ``os-short-id`` :
+
+```
+metadata:
+  name: myvm
+  labels:
+    os-short-id:
+```
+
+The os-short-id label is based on the short OS identifier from [libosinfo](https://libosinfo.org/).
+Libosinfo database can be queried to extract additional information about the operating system using the short-id:
+
+```
+# List all operating systems
+$ osinfo-query os
+
+
+ Short ID             | Name                            | Version | ID
+---------------------------------------------------------------------------------------------------
+...
+ fedora24             | Fedora 24                    | 24        | http://fedoraproject.org/fedora/24
+ fedora25             | Fedora 25                    | 25        | http://fedoraproject.org/fedora/25
+ fedora26             | Fedora 26                    | 26        | http://fedoraproject.org/fedora/26
+...
+
+```
+
+### Use with presets
+
+A Virtual Machine Preset representing an operating system could be applied on any given Virtual Machine using the
```

Comment:
> A Virtual machine Preset suited for a specific guest can then have a label selector to match a present against the `os-short-id` label of a VM.

Along those lines. I'd be explicit thata the preset should math against the os-short-id label.